.circleci/config.yml

```diff
@@ -245,6 +260,7 @@ workflows:
       - test-ios
       - integration-test-ios
       - test-android
+      - trigger-dependent-updates
```

Comment:
I guess this needs to be moved to the `deploy` workflow right? And make sure all steps finish correctly before triggering this.

Reply:
oh yes sorry, I wash in a rush to open the PR and forgot this here